### PR TITLE
text fields, menu layouts are not saved after validation PR

### PR DESCRIPTION
**Pull request by naveen17797**
text fields and menu layouts which are not saved after validating the globals, patch has been made.

### CODE REVIEW

**Comment by teryhill** [comment]
@pri2si17-1997 do you have any Idea why his commits look like this?

File: interface/super/edit_globals.php [hunk 1]
```diff
@@ -269,6 +271,16 @@ function checkBackgroundServices(){
               $boolean = false;
             }
           }
+          elseif ($fldid == "theme_tabs_layout") {
+            //menu array created sine the globals dont have the array
```
Comment (aethelwulffe) [inline interface/super/edit_globals.php]: *Since*

File: interface/super/edit_globals.php [hunk 2]
```diff
@@ -269,6 +271,16 @@ function checkBackgroundServices(){
...
+            $menu_array = array('tabs_style_compact.css', 'tabs_style_full.css');
+            if (in_array($_POST["form_$i"], $menu_array)) {
```
Comment (aethelwulffe) [inline interface/super/edit_globals.php]: Suggestion:
*expression* ? true : false;

**Comment by pri2si17-1997** [comment]
@teryhill maybe he is not doing in one branch and then merging them to a single branch and raising PR. :thinking:

**Pull request merged**